texts/plugins/overview.yml

```diff
@@ -6,5 +6,5 @@ discovery:
 recommender:
   title: Recommender
   description: |
-    Enrich your full text coverage dynamically adding missing bits from CORE
+    Enrich your full text coverage dynamically by adding missing content from CORE
```

Comment:
This description doesn't make sense to me. How do you enrich the full text coverage with Recommender? 🤔 And what missing bits does recommender add?

Reply:
I edited the text without paying much attention to it, it was _CORE services: CORE Recommender and CORE Discovery_ before. I wanted to shorten it.

You can switch 2 names, write _CORE Discovery and CORE Recommender_, would it be more clear?

Reply:
I am not following. 😕

Reply:
I edited the string in the file where you go it but did not pay much attention to the result. I may be mistaken because I see only part of the sentence.

Reply:
What about something like `Get instant recommendations from CORE's database and discover articles of interest from across the network of open access repositories.`

Reply:
`... and journals.` Sounds good!